src/Resources/public/contao_files/templates/smartgear/ce_hyperlink.html5

```diff
@@ -1,3 +1,4 @@
+<?php $this->extend('block_searchable'); ?>
```

Comment:
To be removed, file updated in another PR

Reply:
done